packages/integration-tests/cmr.js

```diff
@@ -126,20 +126,30 @@ const sampleUmmGranule = {
 };
 
 /**
- * Returns true if the concept exists - if the cmrLink
- * returns a 200 and there are entries
+ * Returns true if a granule concept search link returns 200,
+ * false if a 404 is returned.
  *
  * @param {string} cmrLink
- *   CMR URL path to concept, i.e. what is returned from post to cmr task
- *   See `@cumulus/cmrjs/cmr-utils/publishECHO10XML2CMR` for expected URL
+ *   CMR URL search path to granule concept search, e.g
+ *   https://{cmr_url}/search/concepts/{concept_id}
+ *
  * @returns {boolean} true if the concept exists in CMR, false if not
  */
 async function conceptExists(cmrLink) {
-  const response = await got.get(cmrLink, { responseType: 'json' });
+  let response;
+
+  try {
+    response = await got.get(cmrLink);
```

Comment:
It seems like ` { responseType: 'json' }` would still be necessary for the `got` call?

Reply:
Oh, I guess not because we're just inspecting the status code